src/components/Buttons/Buttons.tsx

```diff
@@ -2,12 +2,28 @@ import React from 'react'
 import styled from 'styled-components'
 import { GaramondWoff } from '../../fonts/Garamond.woff'
 import { GaramondWoff2 } from '../../fonts/Garamond.woff2'
+
 export interface ButtonsProps {
   label?: string
   learnabled?: boolean
   disabled?: boolean
   onClick?: () => void
 }
+
+const FirstWrapper = styled.div`
+  display: inline-block;
+  position: relative;
+  background: rgba(252, 70, 43, 0.3);
+`
+const SecondWrapper = styled.div`
+  display: inline-block;
+  background: rgba(21, 140, 177, 0.3);
+  position: absolute;
+  position: relative;
```

Comment:
There is two positions property here, so the last one must override the absolute above.